src/app/src/Map.js

```diff
@@ -16,7 +16,14 @@ class GLMap extends Component {
                     {...this.props.viewport}
                     onViewportChange={this.props.updateViewport}
                     mapboxApiAccessToken={process.env.REACT_APP_MAPBOX_API_KEY}
-                />
+                    mapStyle={'mapbox://styles/alash/cjpconu8b0sas2qn5rx51uog8'}
+                    interactive={false}
```

Comment:
This doesn't appear to disable the zoom via mouse wheel.

![screenshot from 2019-01-09 14 15 49](https://user-images.githubusercontent.com/1014341/50922784-51c34380-1419-11e9-83a0-b2e032caeff7.png)

Reply:
Thanks. Pushed a fixup commit to address this. Ready for another look.